assets/js/modules/adsense/datastore/sites.test.js

```diff
@@ -0,0 +1,218 @@
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( MODULES_ADSENSE )
+							.isFetchingGetSites( fakeAccountID ) === false
+				);
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( fakeAccountID );
+				expect( sites ).toEqual( undefined );
+				expect( console ).toHaveErrored();
+			} );
+		} );
+		describe( 'getSite', () => {
+			beforeEach( () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				// Load data into this store so there are matches for the data we're about to select,
+				// even though the selector hasn't fulfilled yet.
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+			} );
+			it.each( [
+				[ 'www.example.com', fixtures.sites[ 0 ] ],
+				[ 'othersubdomain.example.com', fixtures.sites[ 0 ] ],
+				[ 'www.test-site.com', fixtures.sites[ 1 ] ],
+				[ 'some-other-tld.ie', fixtures.sites[ 2 ] ],
+			] )(
+				'finds the site in this account that matches the domain: %s',
+				async ( domain, expected ) => {
+					const accountID = fixtures.clients[ 0 ]._accountID;
+					const site = registry
+						.select( MODULES_ADSENSE )
+						.getSite( accountID, domain );
+					expect( site ).toEqual( expected );
+				}
+			);
+			it.each( [
+				[ 'non-existent-site.com' ],
+				[ 'some-other-tld.com' ],
+			] )(
+				'returns null when no site matches the given domain: %s',
+				async ( domain ) => {
+					const accountID = fixtures.clients[ 0 ]._accountID;
+					const site = registry
+						.select( MODULES_ADSENSE )
+						.getSite( accountID, domain );
+					expect( site ).toEqual( null );
+				}
+			);
+			it( 'returns null when no site matches the given domain', async () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				const site = registry
+					.select( MODULES_ADSENSE )
+					.getSite( accountID, 'www.non-existent-url.com' );
+				expect( site ).toEqual( null );
+			} );
+		} );
+		describe( 'getCurrentSite', () => {
+			beforeEach( () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				// Load data into this store so there are matches for the data we're about to select,
+				// even though the selector hasn't fulfilled yet.
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+			} );
+			it( 'gets the AdSense site which matches the domain of the current site', async () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				await registry.dispatch( CORE_SITE ).receiveSiteInfo( {
+					referenceSiteURL: 'http://example.com',
+				} );
+				const site = registry
+					.select( MODULES_ADSENSE )
+					.getCurrentSite( accountID );
+				expect( site ).toEqual( fixtures.sites[ 0 ] );
+			} );
+			it( 'returns null when the current site domain does not matches any site', async () => {
```

Comment:
Please add a line break for each suit and case.
```suggestion

			it( 'returns null when the current site domain does not matches any site', async () => {
```